XRTK-Core/Packages/com.xrtk.core/Runtime/Definitions/InputSystem/MixedRealityInputSystemProfile.cs

```diff
@@ -99,6 +99,16 @@ public SystemType GazeProviderType
 
         #region Global Hand Options
 
+        [SerializeField]
+        [Range(.1f, 1f)]
+        [Tooltip("Threshold in range [0, 1] that defines when a hand is considered to be grabing.")]
```

Comment:
And again.

Kinda concerned about the amount of duplicate code. But we can address later